src/test/java/catan/controllers/FunctionalTestUtil.java

```diff
@@ -1,6 +1,9 @@
+// TODO: remove checking of statusCode in each method. (Needs to be approved)
+
 package catan.controllers;
```

Comment:
not good practice to write comments at the begining of the file, better add it at the begining of class body